WindowsServerDocs/troubleshoot/nodes-failover-cluster-vmware.md

```diff
@@ -1,25 +1,27 @@
 ---
 title: Remove node from active failover cluster membership
+ms.topic: troubleshooting
 description: This article addresses the issue of finding nodes removed from active failover cluster membership.
-ms.date: 05/28/2020
 author: Deland-Han
 ms.author: delhan
-ms.topic: troubleshooting
+ms.date: 01/06/2023
 ---
 # Nodes being removed from Failover Cluster membership on VMWare ESX
 
+Applies to Windows Server (All supported versions)
+
 This article addresses the issue of finding nodes removed from active failover cluster membership when the nodes are hosted on VMWare ESX.
 
 ## Symptom
 
-When the issue occurs, you will see in the System Event Log of the Event Viewer:
+You'll see the following in the System Event Log of the Event Viewer when this issue occurs:
 
 ![Event 1135](media/nodes-failover-cluster-vmware/1135.png)
 
 ## Resolution
 
-One specific problem is with the VMXNET3 adapters dropping inbound network packets because the inbound buffer is set too low to handle large amounts of traffic. We can easily find out if this is a problem by using Performance Monitor to look at the “Network Interface\Packets Received Discarded” counter.
+One specific problem is with the VMXNET3 adapters dropping inbound network packets because the inbound buffer is set too low to handle large amounts of traffic. You easily can find out if this is a problem by using Performance Monitor to look at the “Network Interface\Packets Received Discarded” counter.
```

Comment:
```suggestion
One specific problem is with the VMXNET3 adapters dropping inbound network packets because the inbound buffer is set too low to handle large amounts of traffic. You easily can find out if packets are being drop by using Performance Monitor to look at the “Network Interface\Packets Received Discarded” counter.
```